src/story_gui.cpp

```diff
@@ -712,8 +712,9 @@ struct StoryBookWindow : Window {
 
 			switch (ce.pe->type) {
 				case SPET_TEXT:
-					y_offset = DrawStringMultiLine(ce.bounds.left, ce.bounds.right, ce.bounds.top - scrollpos, ce.bounds.bottom - scrollpos,
+					DrawStringMultiLineWithClipping(ce.bounds.left, ce.bounds.right, ce.bounds.top - scrollpos, ce.bounds.bottom - scrollpos,
 						ce.pe->text.GetDecodedString(), TC_BLACK, SA_TOP | SA_LEFT);
+					y_offset = ce.bounds.bottom - scrollpos;
```

Comment:
It's not even used, is it?
```suggestion
```

Reply:
Good point, I thought it was cumulative, but 710 just replaces the value anyway.